apps/glusterfs/pendingop.go

```diff
@@ -28,23 +28,33 @@ import (
 type PendingOperationType int
 
 const (
+	// These values are persisted and we should never change the mapping. Even
+	// if an operation type is deprecated, use a new one and retire the number
+	// associated with it.
+
 	OperationUnknown PendingOperationType = iota
 	OperationCreateVolume
 	OperationDeleteVolume
 	OperationExpandVolume
 	OperationCreateBlockVolume
 	OperationDeleteBlockVolume
-	OperationExpandBlockVolume
 	OperationRemoveDevice
 	OperationCloneVolume
 	OperationBrickEvict
+	OperationExpandBlockVolume
+	// If you have any edit except directly above this line it is probably a
+	// mistake. Read the top comment about renumbering the enums.
 )
 
 // PendingChangeType identifies what kind of lower-level new item or change
 // is being made to the system as part of a higher-level pending operation.
 type PendingChangeType int
 
 const (
+	// These values are persisted and we should never change the mapping. Even
+	// if a change type is deprecated, use a new one and retire the number
+	// associated with it.
+
```

Comment:
Needed an empty line here because the comment was being taken as a doc for the exported const OpUnknown.